src/main/flight/imu.c

```diff
@@ -687,7 +679,7 @@ static void imuCalculateEstimatedAttitude(timeUs_t currentTimeUs)
         gyroAverage[axis] = gyroGetFilteredDownsampled(axis);
     }
 
-    const bool useAcc = imuIsAccelerometerHealthy(acc.accADC); // all smoothed accADC values are within 20% of 1G
+    const bool useAcc = imuIsAccelerometerHealthy(); // all smoothed accADC values are within 10% of 1G !! Used to say 20% but was actually 10%
```

Comment:
It doesn't matter what the comment used to say. It's only important that the comment is correct
```suggestion
    const bool useAcc = imuIsAccelerometerHealthy(); // all smoothed accADC values are within 10% of 1G
```

Reply:
Yes, for the end product, definitely.